docs/storage-drivers/inmemory.md

```diff
@@ -1,3 +1,7 @@
+page_title: In-memory storage driver
```

Comment:
Having these make the docs hard to read in the repo. Is there any other way to specify these not to be so obtrusive?